pkg/scalers/splunk_scaler.go

```diff
@@ -0,0 +1,196 @@
+package scalers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/go-logr/logr"
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+
+	"github.com/kedacore/keda/v2/pkg/scalers/scalersconfig"
+	"github.com/kedacore/keda/v2/pkg/scalers/splunk"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+const (
+	splunkDefaultHTTPTimeout = 10 * time.Second
```

Comment:
KEDA sets this global timetout: https://keda.sh/docs/2.14/operate/cluster/#http-timeouts do we need a specific one for Splunk?